client/package.json

```diff
@@ -28,8 +28,8 @@
     "bpmn-js": "^13.0.0",
     "bpmn-js-properties-panel": "^1.22.1",
     "bpmn-moddle": "^8.0.1",
-    "bpmn-js-tracking": "0.2.0",
-    "camunda-bpmn-js": "^2.3.1",
+    "bpmn-js-tracking": "0.3.0",
+    "camunda-bpmn-js": "^2.5.0",
```

Comment:
Let's make this bump explicit, and include changes pulled in via `camunda-bpmn-js` bump in the `CHANGELOG`.